.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should not be set in the project settings but in user settings; please remove this file